docs/pluggable_storage.rst

```diff
@@ -57,6 +57,25 @@ AzurePinotFS requires the following configs according to your environment:
 
 adl.accountId, adl.authEndpoint, adl.clientId, adl.clientSecret
 
+Sample Controller Config
```

Comment:
Can we format these configs as JSON?

Reply:
To format as JSON, it requires {}, and I don't want to confuse our users, as just the flat config without the braces is what they need to put in their configuration file

Reply:
Oh I see, this is not JSON config, but apache PropertiesConfiguration file. In that case, this is fine. Could you please add instruction on how this file is passed to `pinot-server`? Also, please make sure that the output html page is formatted correctly.

Reply:
I verified that the output is formatted. Can you clarify what you mean by instruction on how the file is passed? It's the config file that users can save and input to startServer, so we don't do anything special (they essentially just need to declare this path for their config)